sdk/analytics/src/main/java/com/klaviyo/analytics/networking/requests/JSONUtil.kt

```diff
@@ -0,0 +1,9 @@
+package com.klaviyo.analytics.networking.requests
+
+import org.json.JSONObject
+
+internal object JSONUtil {
+
+    fun JSONObject.getStringNullable(key: String): String? =
```

Comment:
Maybe we already have something like this? We don't do a ton with responses currently so my thought was this could be helpful for dealing with malformed responses

Reply:
The only JSON decoding utils we have are related to decoding [network requests](https://github.com/klaviyo/klaviyo-android-sdk/blob/4a6f806064d1e24f1f4be1cefcdc87852b87cfa9/sdk/analytics/src/main/java/com/klaviyo/analytics/networking/requests/KlaviyoApiRequestDecoder.kt#L14) or [profile](https://github.com/klaviyo/klaviyo-android-sdk/blob/99a60a65bcb8d20cea5c81014ed3c69e892dc1bc/sdk/analytics/src/main/java/com/klaviyo/analytics/state/PersistentObservableProfile.kt#L42) objects from the persistent store. More tailor made, and simply catch JSONExceptions to detect unexpected JSON formats. 

So yeah this is a good addition